plugins/net_plugin/net_plugin.cpp

```diff
@@ -3774,34 +3773,15 @@ namespace eosio {
 
          if (fork_db_add_result == fork_db_add_t::appended_to_head || fork_db_add_result == fork_db_add_t::fork_switch) {
             ++c->unique_blocks_rcvd_count;
-            fc_dlog(logger, "posting incoming_block to app thread, block ${n}", ("n", ptr->block_num()));
-            my_impl->process_blocks();
-
+            fc_dlog(logger, "post process_incoming_block to app thread, block ${n}", ("n", ptr->block_num()));
```

Comment:
Not sure if this log is still appropriate as process_blocks has been moved to producer_plugin.

Reply:
I think it is useful debug output here.